image_view/include/image_view/disparity_view_node.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef IMAGE_VIEW__DISPARITY_VIEW_NODE_HPP_
```

Comment:
For all files you made, can you add copyright headers? This removed ambiguity

Reply:
Done. For those that already had BSD headers, I added a second one for Apache 2.0. For those that had Apache 2.0 headers, I combined them with a new one. For those that had no headers (new files), I just added the Apache 2.0 header.